python_modules/dagster/dagster_tests/cli_tests/workspace_tests/hello_world_file_in_directory/hello_world_repository.py

```diff
@@ -1,6 +1,7 @@
 # type: ignore
 
 from dagster import repository
+
```

Comment:
rm me

Reply:
ruff did this

Reply:
`ruff` also failed on this PR though https://buildkite.com/dagster/dagster/builds/49940#01867af6-7040-42a1-ae56-aee757a65792

Reply:
weird